src/theme/Navbar/index.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import NavbarLayout from "@theme/Navbar/Layout";
+import NavbarContent from "@theme/Navbar/Content";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import Logo from "../../../images/polykey-logo-light.svg";
+export default function Navbar() {
+  const [dynamicClass, setDynamicClass] = React.useState(false);
+  const [email, setEmail] = React.useState("");
+  return (
+    <NavbarLayout>
+      <div className="container">
+        <div className="header_main">
+          <div className="header_logo">
+            <img src="https://uploads-ssl.webflow.com/6098c4e3255a0a67635c1de5/6098c4e3255a0ac4e35c1e05_poly-key-logo.svg" />
+          </div>
+          <div className="header_menu">
+            <ul>
+              <li>
+                <a href="https://polykey.io/blog">Blog</a>
+              </li>
+              <li>
+                <a href="https://polykey.io/docs">Docs</a>
+              </li>
+              <li>
+                <a href="https://polykey.io/download">Download</a>
+              </li>
```

Comment:
These links should come from the `navbar.items` configuration. It shouldn't be hardcoded here.